TaxonomyAPIs new branch(07-08) (#9002)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/termstore-localizedlabel.md

---
title: "localizedLabel resource type"
description: "Describes the facet for labels of terms in the termStore"
author: mohitpcad
ms.author: mopathak
localization_priority: Normal
ms.prod: "sharepoint-taxonomy"
doc_type: resourcePageType
---

# localizedLabel resource type

Namespace: microsoft.graph.termStore

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

The **microsoft.graph.termStore.localizedLabel** resource represents the term-label of a [microsoft.graph.termStore.term] used in the [microsoft.graph.termStore.store].

Identifies the labels associated with a given term. Inherits from [microsoft.graph.termStore.localizedName].

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |isDefault|Boolean|Whether label is default label or not|

Reviewer: ```suggestion
|isDefault|Boolean|set to **True** if the label is default|
```

Reviewer: suggested wording for documentation

---
title: "localizedLabel resource type"
description: "Represents the label for a term in the term store."
author: mohitpcad
localization_priority: Normal
ms.prod: "Sharepoint"
doc_type: resourcePageType
---

# localizedLabel resource type

Namespace: microsoft.graph.termStore

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents the label for a [term] in the [term store].

Reviewer 2: Global: Term store is two words. When used in text, it needs to be "term store", not "termstore". References to the namespace can remain one word, lower camel cased. Please update occurrences in the text, including the links, to use two words.
